recipes-debian/squashfs-tools/squashfs-tools_debian.bb

```diff
@@ -0,0 +1,47 @@
+# base recipe: meta/recipes-devtools/squashfs-tools/squashfs-tools_git.bb
```

Comment:
mksquashfs requires libgcc so it would be nice to add libgcc package in RDEPENDS.

```
root@qemuarm64:~# mksquashfs /etc etc.squashfs
Parallel mksquashfs: Using 1 processor
Creating 4.0 filesystem on etc.squashfs, block size 131072.
libgcc_s.so.1 must be installed for pthread_cancel to work
Aborted
```